docs/tutorials/08_cvar_optimization.ipynb

```diff
@@ -217,22 +213,18 @@
     "def callback(i, params, obj, stddev, alpha):\n",
     "    # we translate the objective from the internal Ising representation\n",
     "    # to the original optimization problem\n",
-    "    objectives[alpha] += [-(obj + offset)]\n",
+    "    objectives[alpha].append(np.real_if_close(-(obj + offset)))\n",
     "\n",
     "\n",
     "# loop over all given alpha values\n",
     "for alpha in alphas:\n",
     "\n",
-    "    # initialize CVaR_alpha objective\n",
-    "    cvar_exp = CVaRExpectation(alpha, PauliExpectation())\n",
-    "    cvar_exp.compute_variance = lambda x: [0]  # to be fixed in PR #1373\n",
-    "\n",
     "    # initialize VQE using CVaR\n",
```

Comment:
This comment here says VQE - would this be better now as SamplingVQE?

The title of this section (where I cannot comment easily) still has VQE and more likely should be SamplingVQE now

> ## Minimum Eigen Optimizer using VQE